perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -523,7 +533,15 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 			if (parms.contains("facility")) {
 				if (parms.contains("user")) {
-					if (parms.contains("member") && parms.contains("resource")) {
+					if (parms.contains("member") && parms.contains("resource") && parms.contains("group")) {
```

Comment:
We always use more levels of 'if clause' when asking about parameters. From my point of view it is more human readable. Something like: 
if(parms.contains("member") && parms.contains("resource")) {
___if(parms.contains("group") {
______...method with group
___} else {
______...method without group
___}
}